perf/micro/immediate-scheduler/observable/from-array.js

```diff
@@ -8,7 +8,7 @@ module.exports = function fromArray(suite) {
     }
 
     var oldFromArrayWithImmediateScheduler = RxOld.Observable.fromArray(args, RxOld.Scheduler.immediate);
-    var newFromArrayWithImmediateScheduler = RxNew.Observable.fromArray(args);
+    var newFromArrayWithImmediateScheduler = RxNew.Observable.from(args);
```

Comment:
I ran this a few times, and didn't at all see what you're seeing. RxJS Next was still faster, with no noticeable difference in speed from what's in master. Can you run again to confirm? I have serious doubts that making this function polymorphic and one assertion of `isArray` would make such a large difference.

I'm going to go ahead and merge for now. It's easy enough to add `fromArray` back in if we need to.